### DISCUSSION TITLE
Final project - ideas

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#25] [Homework]
Write a short description of for __two__ ideas about a culture, or knowledge field you will represent for your final project:

- what is it? what are the different lenses through which we can look at it?
- why is it important?  why is it relevant? to you? to others?
- how are you going to represent it? what kind of media? what kinds of interaction?

**Comment by chaerin-d** [comment]
[Group with Ambra and Robbin]

We decided to create a promotional site about composer Debussy. It will be interesting to see a combination of classic and modern technology, that these days, most of musicians create goods, interactive designs...etc for their promotion. Especially, for our group, we all like classical music, and through this project, we want to learn more about Debussy and develop a connection between classical music and people nowadays. Currently, we are thinking of using some techniques for creating maps(for regional connection of composer), playlists, and designing albums...

**Comment by bhornik** [comment]
(Beckett - Alex - Liz)

1) Our concept is to create a collection of album covers to create a visual timeline of cultural and political trends through a vast array of popular music/artists over a 60-80 year period. The main lenses through which we will look at it are politics, religion, sexuality, and popular culture. 

2) A tool to visually track cultural trends, and learn more about music’s essential role in various cultural and political movements. It is relevant to us because we, and a majority of the people we know, consume music on a daily basis and are both consciously and unconsciously affecting the world around us. 

3) The main page is purely a visual timeline and various movements/trends can loosely or very clearly be tracked through the album artworks presented. The main media is images, but hyperlinks (both text and highlighted images) throughout the page can be pressed on to learn more about specific albums, decades, or movements. The page is navigated via links and images and has no specific route, but instead encourages random discovery.

**Comment by Caythian** [comment]
For the final project, I want to develop a website dedicated to the breathtaking natural landscapes we see in “Zelda: Tears of the Kingdom” (Zelda). But I will also use resources from “Zelda: Breath of the Wild” because of the massive research that has already been done by other passionate gamers and researchers. This website is going to 1) delve into the landscapes themselves, create a visual guide for players who are interested in exploring the game from a visual perspective, 2)explore the relationship between the landscapes in games and the landscapes in the real world, and 3) offer insight about how virtural landscape could be used to improve human’s well-being and our understanding of the environment.

1)Guide to the natural landscapes in Zelda:
First, I want to include a general introduction discussing the natural landscapes in Zelda. It would be dealing with the uniqueness of this particular open-world game about its landscape, as well as the reason for me to do this project.
Second, a more detailed guide to the landscapes will be supplemented by implementing a detailed, interactive map of the game's world, highlighting specific scenic locations. Users could click on these spots to learn more about this place and the region. Specifically, how Link could reach the location quickly, and what attractions are nearby. However, this part won’t be the very focus of the project, therefore it would only include a few locations as examples; my next point will be explaining why. 
Lastly, a more interactive section will be provided, allowing the user to upload their favorite place in the Game. By doing so, this website will not be only about the developer’s perspective of the game but a public one as well. 
2) Landscapes in Game vs. Landscapes in Reality:
The first thing I want to do is about Zelda itself- comparing the landscape in this game with the landscapes in reality. Are there locations in this game that are inspired by reality? 
Next, I want to discuss the value of featuring beautiful landscapes in the game that might inspire protection of the real natural landscapes. For instance, do we feel the same kind of urge and excitement in reality as we would feel when exploring an unknown area in the game? Could such psychological responses be a hidden intuition of human beings? 
The previous point could lead to the discussion of why game developers develop a realistic landscape in games. It can include interviews with the game's designers or psychological experts on how landscapes are deliberately crafted to create certain feelings or experiences.
3) In-game Landscapes and their Impacts:
To begin this section, I intend to introduce my personal experience of playing Zelda and how it impacted my mental health. It could introduce research about the psychological influence, the positive, as well as negative impact of the virtual landscape on people. For instance, does a virtual landscape create anxiety because of its surreality? Or does such a landscape actually help people overcome their fears in reality?
Next, I would like to discuss if an in-game landscape would raise people’s concerns about environmentalism. Experiencing in-game landscapes can sometimes influence a player's real-world behavior, such as increasing appreciation for nature, promoting conservation efforts, or influencing travel choices.
Lastly, in therapeutic settings, virtual landscapes can be used as tools for meditation, exposure therapy for phobias, or as controlled environments for rehabilitation. Could the open-world games be a virtual sandbox where people reflect their psychological space into the objects, in the future? This part will delve into the potential of games in the future, in particular in education (language learning, physics, design, etc). For the three points mentioned in number 3, the main challenge is how to prevent my website from turning into textual research. To combat this challenge, I would like to include visual examples of the games I recommend. 

To be honest, when I am looking back at the statements, I think the last part could already constitute an interesting website:) And maybe Zelda would just be the inspiration :)

**Comment by zbyiii** [comment]
For the final project, our group is going to work on the topic of trees in Berlin. In this project, we will investigate the tress in Berlin from different aspects, such as species, distribution in the city, history of the green space, tress as part of the community and so on. 

Berlin is one of the greenest cities in Europe. And there are many different types of trees in Berlin. The trees have a long history of witnessing the changes of the city. However, in recent years, as the city is experiencing drought it is hard for trees to get enough water. There are organizations call on citizens to water the trees, and there is an interactive map on its website. We would like to make a website to collect the normally seen species of trees and raise the awareness of preserving the green space. 

We believe as human are part of the nature, we should preserve the nature and interact with it. Therefore, it is important to know the environment and the non-human community around us. Also, we want to make this website to provide information to those who are also interested in nature. 

We will include different elements on the website to show the information we gathered and add some interactive part. First, we will make a digital archive showing the human representation of trees. Also, we will add a map component to present the density, diversity, and watering status of the trees. Besides, we will include a case study about the history of trees in Tiergarten. 

Interaction:  Map component, click to show legend of picture/ text

**Comment by el-dah** [comment]
One idea: Ethio-/Eritrean art, beauty, and ritual
• exploration of cultural identity through Habesha (Ethiopian, Eritrean) art, beauty practices, and rituals
• focus on body adornment, ceremonial objects, and traditional practices
		• highlights cultural resilience and challenges stereotypes
		• relevant to me for building on existing work; offers others a deeper understanding of some indigenous expression
• multimedia digital installation with visual art, videos, interviews, and interactive digital elements

Other idea: minority representation in club culture

• focus on how women, queer, Black, and Brown people especially use nightlife for self-expression and activism
• examines club culture’s role in positive / affirmative identity formation as well as its limitations
		• raises awareness about barriers marginalized groups face in and outside of nightlife (within a confined scope)
		• relevant for discussing social justice and cultural expression; encourages dialogue on inclusivity
• immersive club style audiovisual installation with soundscapes, fashion displays, and archival footage.
		• optional: incorporates interviews / oral histories

**Comment by Ristar-10** [comment]
two ideas for final project

Berlin Techno Music
- Berlin's influence on techno music, how it came to be so popular, evolution, how it's tied to Berlin's identity
- important part of Berlin and club culture here, could also examine club culture a bit and how it's become a space for self expression etc. 
-I can learn more about it as I've grown to appreciate the techno music while I've been here, relevant to others who are involved in the techno/club scene, looking to know more etc. 
- represented though digital environments of club spaces, interacting to play music, displaying history and other information in a thematic way that fits with the rest

Magic the Gathering
- could focus on gameplay strategy and lore as well as the history behind it
- first trading card game, groundbreaking in a way
- relevant to me as a player, I can learn more about it while working on this project, relevant to other players in the community, people interested in card/game design.
- represented with a lot of colors and a magical/fantasy element with images and such like the game is, interaction through clicking to flip over cards to reveal information maybe
- archives of card/set releases, tournament information, different game formats, deck combinations, etc.

**Comment by jkaveh-dr** [comment]
1. Continuing working on Queer Utopias - i would continue building the site as a form of documentation. I would map the queer diaspora, between Berlin, NYC, and Sri Lanka, because these are the scenes I have personal experience. There would also be a specific focus on the trans narrative, highlighting radical trans spaces to provide information for those seeking that kind of safe and affirming space.

2. creating an interactive artwork where I use hypertext and prose as the medium - I was really inspired by the Hypertext as Subversive and would be interested in employing a similar ideology to convey my experiences of trans-ness and gender liberation.

3. I will be helping Marianna create her site focusing on Tijuana because her concept inspires me and I am excited to work on it :)

**Comment by Cate-Hackett** [comment]
Forgot to write the two ideas last week- here they are for reference!
1. PERSONAL COMMUNiCATION TECH ARCHIVE - 
The most influential personal (communication) technology devices - the ways they have changed through history, changed social behavior and society, and the different products, allowances, interactions with themselves they have elicited by shape shifting. (also super interesting because I’ve been to quite a few (international) tech museums and there are kind of different narratives for who built the first influential computers and what not, so it’d be interesting to figure out what could be a good low-level individual influence social personal tech archive

2. History, culture and shape shifting of GLASSES ! what they’ve looked like (maybe in different cultures, what they originally looked like)
